source/CommonJobs/CommonJobs.MVC.UI/Content/Evaluations/evaluations.css

```diff
@@ -1185,6 +1190,7 @@ tr.evaluation-generated td {
   position: fixed;
   right: 0;
   left: 0;
+  display: none;
```

Comment:
The loader-container needs to be visible by default so that until the app loads and the javascript starts to run, the loader is shown.